packages/server/src/api/notifRoutes.ts

```diff
@@ -0,0 +1,112 @@
+import { ServerRoute, ResponseObject } from "@hapi/hapi";
+import { NotifBody } from "@template/common";
+import { NotifModel } from "../entity/NotifModel";
+import { NotifPayload } from "../joi";
+import * as dotenv from "dotenv";
+import * as webPush from "web-push";
+import { DeepPartial } from "typeorm";
+
+// configure env vars for VAPID
+dotenv.config();
+
+// if env vars not found, then throw an error early
+if (!process.env.EMAIL || !process.env.PUBLICKEY || !process.env.PRIVATEKEY) {
+  throw new Error(
+    "please add a .env file with keys+email in packages/server. ask alex/eddy for deets."
+  );
+} else {
+  webPush.setVapidDetails(
+    process.env.EMAIL,
+    process.env.PUBLICKEY,
+    process.env.PRIVATEKEY
+  );
+}
+
+export const notifRoutes: ServerRoute[] = [
+  {
+    // endpoint to get the VAPID public key
+    method: "GET",
+    path: "/api/v1/notifications/credentials",
+    handler: (request, h) => {
+      return JSON.stringify(process.env.PUBLICKEY);
+    },
+  },
+  {
+    method: "POST",
+    path: "/api/v1/notifications/register/{user_id}", // TODO:   make this not a param for the users lmaoooooooo soI don't spam alex with sugondese
```

Comment:
Am I correct that we'd get the user_id of the currently logged in user instead (once user auth is implemented)?

Reply:
yeet. dont want eddy registering his phonenumber for me